src/dev-starter.ts

```diff
@@ -33,7 +34,13 @@ export class DevStarter {
 
     const host = 'https://' + this.lxrConfig.host;
     const accessTokenHash = accessToken ? `#access_token=${accessToken}` : '';
-    const baseLaunchUrl = `${host}/${this.lxrConfig.workspace}/reporting/dev?url=${urlEncoded}`;
+    let workspace = this.lxrConfig.workspace
```

Comment:
Still taking the workspace from `lxrConfig` is for backwards compatibility, there should always be a workspace in the JWT, shoudn't it?